README.md

```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
 
 Instead, the projects (i.e., for TYPO3 installations) need to have the security checks.
 
+## Documentation rendering
+
+In order to render documentation, first of all, clone repository
```

Comment:
Can we link the ReST documentation from the README? (IIRC, GitHub can render ReST.)

Reply:
OK I will check this one. Maybe we can add it in separate PR with new issue, and merge this one first?